Services/AuthenticationService/Controllers/AuthenticationController.cs

```diff
@@ -31,7 +33,9 @@ public string Login([FromServices] ILoginCommand command, [FromBody] UserEmailPa
         {
             var result = command.Execute(user);
 
-            return result.Result;
+            var userToken = result.Result;

```

Comment:
I suggest to change the name of result. What type it is? Why does result have result properties. It isn't understandable

Reply:
It's Task<UserToken>, what are suggestions?

Reply:
Fixing in next commit